src/pages/upload/index.tsx

```diff
@@ -312,44 +318,40 @@ const Upload: React.FC<
                     <span className="text-blue-600 underline">browse</span>
                   </span>
                 </span>
-                <VideoUploader
-                  dispatch={dispatch}
-                  setLessonMetadata={setLessonMetadata}
-                />
+                <VideoUploader dispatch={dispatch} />
               </label>
             </div>
           </div>
-          {fileUploadState.files.map((file) => {
-            if (file.percent < 100) {
-              return (
-                <div>
-                  {file.file.name} - {file.message} - {file.percent}%
-                </div>
-              )
-            } else {
-              return null
-            }
```

Comment:
Instead of momentarily displaying the file upload state which causes a bit of layout shifting, I embed the percentages in the markup below.